.htaccess

```diff
@@ -8,10 +8,18 @@ RedirectMatch 302 \
 	^/forum/security/?$ \
 	https://forums.classicpress.net/c/team-discussions/classicpress-security
 
+RedirectMatch 302 \
+	^/faq-support/
+	https://link.classicpress.net/faq-support/
+
 RedirectMatch 302 \
 	^/plugin-compatibility/?$ \
 	https://docs.classicpress.net/faq-support/#will-my-current-plugins-and-themes-work-in-classicpress
 
+RedirectMatch 302 \
+	^/docs \
```

Comment:
A few things:

- All _source_ links (like this line) should end in `/?$`, this will ensure they work with or without trailing slashes
- All links that points to `docs.classicpress.net` should probably start with `/docs` (I know that we haven't always done this with previous links, but we might as well start now with the new ones)
- Let's sort all of these entries by the source / incoming URL pattern

Reply:
>     * All links that points to `docs.classicpress.net` should probably start with `/docs` (I know that we haven't always done this with previous links, but we might as well start now with the new ones)

Actually I'm not sure about this one, I just saw that I made a different suggestion here: https://github.com/ClassicPress/ClassicPress/pull/516#discussion_r341447070

What do you think @mattyrob ?

Reply:
I guess it make make more sense for future to bundle links into sub-folder groups - so `docs/` could be added to in future with things like `support/` and perhaps even `forums/`.